backend/forms/models/form.py

```diff
@@ -89,5 +90,11 @@ def validated_fields(self):
         return [Field.model_validate(field_dict) for field_dict in self.enriched_fields]
 
     @property
-    def survey(self):
-        return self.event.surveys.filter(languages=self).get()
+    def survey(self) -> Optional["Survey"]:
```

Comment:
```suggestion
    def survey(self) -> Survey | None:
```
Add a `from __future__ import annotations` to defer annotation evaluation so you don't need forward references

Reply:
:cat::+1: TIL

Reply:
I should note here that `from __future__ import annotations` _may_ have unintended side effects with some libraries, etc. that attempt to introspect futured annotations (since they will not have been evaluated into actual objects).

Reply:
Okay so let's be careful about not using this with Pydantic yet